INSTALL.md

```diff
@@ -0,0 +1,43 @@
+# Installing
+
+You need to use WSL or a GNU/Linux distro to build this, a Debian based distro is reccommened. Directions can be found [here](https://learn.microsoft.com/en-us/windows/wsl/install).
+
+## Setting up file structure
+First, download the zip file for this directory and extract it somewhere you will remember.
+### Install Devkitpro
+These instructions are for debian based distributions or WSL. Installation instructions for devkitpro on other distributions can be fount [on their website](https://devkitpro.org/wiki/Getting_Started).
+```bash
+wget https://apt.devkitpro.org/install-devkitpro-pacman
+chmod +x ./install-devkitpro-pacman
+sudo ./install-devkitpro-pacman
+```
+and then install the gba libs:
+```bash
+sudo dkp-pacman -S gba-dev
+```
+and then export the environment values
+```bash
+export DEVKITPRO=/opt/devkitpro
+export DEVKITARM=/opt/devkitpro/devkitARM
+```
+### Installing tools
+Donwload the main tools from [here](https://cdn.discordapp.com/attachments/960983413890908220/960983654220308502/tools.7z?ex=65e771c5&is=65d4fcc5&hm=47aa4fe20beb4d9ab203832eeec680b8b3f70dc6c92eab55a062ce61815490a9&).
```

Comment:
It's not a good idea to host tools on discord and then link them here. Instead the user should probably just install those tools themself. Agbcc can be found here https://github.com/pret/agbcc. As for gbafix, I am not sure where I got that from but it is also part of devkitpro, so perhaps we should just use that instead.